src/base/search/searchdownloadhandler.cpp

```diff
@@ -40,7 +40,7 @@ SearchDownloadHandler::SearchDownloadHandler(const QString &siteUrl, const QStri
     , m_downloadProcess {new QProcess {this}}
 {
     m_downloadProcess->setEnvironment(QProcess::systemEnvironment());
-    connect(m_downloadProcess, static_cast<void (QProcess::*)(int)>(&QProcess::finished)
+    connect(m_downloadProcess, static_cast<void (QProcess::*)(int, QProcess::ExitStatus)>(&QProcess::finished)
```

Comment:
Doesn't current minimal supported Qt version allow to use qOverload?

Reply:
indeed, changed.